wire-ios-system/Source/Logging/WireLogger.swift

```diff
@@ -16,84 +16,6 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-public struct WireLogger: LoggerProtocol {
+import WireFoundation
 
-    private static var provider = AggregatedLogger(loggers: [
-        SystemLogger(),
-        CocoaLumberjackLogger()
-    ])
-
-    public let tag: String
-
-    // MARK: - Initialization
-
-    public init(tag: String) {
-        self.tag = tag
-    }
-
-    // MARK: - LoggerProtocol
-
-    public var logFiles: [URL] {
-        Self.provider.logFiles
-    }
-
-    public func addTag(_ key: LogAttributesKey, value: String?) {
-        Self.provider.addTag(key, value: value)
-    }
-
-    public func debug(_ message: any LogConvertible, attributes: LogAttributes...) {
-        guard shouldLogMessage(message) else { return }
-        Self.provider.debug(message, attributes: finalizedAttributes(attributes))
-    }
-
-    public func info(_ message: any LogConvertible, attributes: LogAttributes...) {
-        guard shouldLogMessage(message) else { return }
-        Self.provider.info(message, attributes: finalizedAttributes(attributes))
-    }
-
-    public func notice(_ message: any LogConvertible, attributes: LogAttributes...) {
-        guard shouldLogMessage(message) else { return }
-        Self.provider.notice(message, attributes: finalizedAttributes(attributes))
-    }
-
-    public func warn(_ message: any LogConvertible, attributes: LogAttributes...) {
-        guard shouldLogMessage(message) else { return }
-        Self.provider.warn(message, attributes: finalizedAttributes(attributes))
-    }
-
-    public func error(_ message: any LogConvertible, attributes: LogAttributes...) {
-        guard shouldLogMessage(message) else { return }
-        Self.provider.error(message, attributes: finalizedAttributes(attributes))
-    }
-
-    public func critical(_ message: any LogConvertible, attributes: LogAttributes...) {
-        guard shouldLogMessage(message) else { return }
-        Self.provider.critical(message, attributes: finalizedAttributes(attributes))
-    }
-
-    // MARK: - Private Helpers
-
-    private func shouldLogMessage(_ message: LogConvertible) -> Bool {
-        return !message.logDescription.isEmpty
-    }
-
-    private func finalizedAttributes(_ attributes: [LogAttributes]) -> LogAttributes {
-        var finalizedAttributes = flattenArray(attributes)
-
-        if !tag.isEmpty {
-            finalizedAttributes[.tag] = tag
-        }
-
-        return finalizedAttributes
-    }
-
-    // MARK: Static Functions
-
-    public static var logFiles: [URL] {
-        provider.logFiles
-    }
-
-    public static func addLogger(_ logger: LoggerProtocol) {
-        provider.addLogger(logger)
-    }
-}
+public typealias WireLogger = WireFoundation.WireLogger
```

Comment:
that might be confusing, though a good idea to have a default logger here. Does it have to be public?
maybe `WireLoggerDefault`?

Reply:
This code has been removed, so the comment is outdated.